t/014-bugs.t

```diff
@@ -593,12 +593,12 @@ $s
     }
 --- pipelined_requests eval
 ["GET /test", "GET /test", "GET /test"]
---- response_body eval
-["0", "0", "0"]
+--- response_body_like eval
+[qr/^(0|3|6)$/, qr/^(1|4|7)$/, qr/^(2|5|8)$/]
```

Comment:
Please do not use capturing groups in such regexes. We should use `(?: )` instead to avoid useless submatch capturing. Please address other similar places in the tests. Also, it seems that it's simpler to just use `[036]` instead of `(?:0|3|6)` here, for example?